pf/internal/plugin/provider_server.go

```diff
@@ -251,9 +251,8 @@ func (p *providerServer) Configure(ctx context.Context,
 		return nil, err
 	}
 
-	p.keepSecrets = req.GetAcceptSecrets()
```

Comment:
D'you have an idea what req.GetAcceptSecrets() is about? Maybe this is about supporting older versions of Pulumi CLI that didn't support secrets?

Reply:
It is to support older CLI versions that don't support secrets.

See https://github.com/pulumi/pulumi/commit/53c31344fa9d0787198ec08dd76cdfbcd464a874